plugins/processors/gpuattributes/processor.go

```diff
@@ -253,3 +211,13 @@ func (d *gpuAttributesProcessor) filterGpuMetricsWithoutPodName(metrics pmetric.
 		return dps.Len() == 0
 	})
 }
+
+func dropResourceMetricAttributes(resourceMetric pmetric.ResourceMetrics) {
+	serviceNameKey := "service.name"
+	attributes := resourceMetric.Resource().Attributes()
+	serviceName, exists := attributes.Get(serviceNameKey)
+
+	if exists && (serviceName.Str() == "containerInsightsNeuronMonitorScraper" || serviceName.Str() == "containerInsightsDCGMExporterScraper") {
```

Comment:
Why the second condition?

Reply:
we only want to drop it from the metrics scrapped by the neuron and dcgm scrappers. These header should exist in neuron and dcgm resource metric attributes